### PR TITLE
Feature: styling

### PR DESCRIPTION
**Pull request by Regiss05**
In this feature we:
- added menu style
- added header style
- added stories style
- added about style

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  padding: 3rem 25px;
+  line-height: 1.2;
+}
+
+.header-title h1 {
+  color: #ff6b00;
+  font-size: 50px;
+  margin: 0;
+  padding: 0;
+}
```
Comment (thecodechaser) [inline main.css:32]: - [ ] Good job working on the styles but still the design is little different from the requirements. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please place some padding between the toolbar (or header) and the hero section So your hero section covers almost 100% of the screen, Follow the design on the template as much as possible.

Please check the images below for reference
Your design

InkedScreenshot (42)

Figma design

Screenshot (43)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            <h1>Hey there. I'm Regiss</h1>
+            <h3>I'am software developer</h3>
+            <p>I can help you build a product. feature or website look through some of my work and experience! if you
+                like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            <div class="logo-plat">
+                <img src="./Images/github.svg" alt="github" />
+                <img src="./Images/linkedin.svg" alt="linkedin" />
+                <img src="./Images/Tert.svg" alt="Tert" />
+                <img src="./Images/twitter.svg" alt="twitter" />
+                <img src="./Images/Medium.svg" alt="Medium" />
```
Comment (thecodechaser) [inline index.html:31]: - [ ] Please wrap your social media links in list, You can use `<ul>` list. When we have similar elements in HTML it's  a good practice to wrap them in a list and it also makes sure our code is semantic.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @Regiss05 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a title and summary. :heavy_check_mark:
- Code is clean. :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._
- [ ]  Please place some padding between your header and hero section so your hero section covers almost 100% of the screen. Check the comments under the review.
- [ ]  Please wrap your social media links in a list, Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ]  For your commit messages please use present tense and use camel case
Bad Example: "added background image"
Good Example: "Add background image"

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Regiss05** [comment]
- Add padding to the header
- Add social media links list
- commit message change

File: index.html [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./main.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav-bar">
+            <div>Regiss M.</div>
```
Comment (prolajumokeoni) [inline index.html:15]: ### OPTIONAL
- [ ] Kindly change div to h1 tag, the h1 tag would be better since this is a text 🤗

**Review by prolajumokeoni — APPROVED** [review]
Hi @Regiss05 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Regiss05) [reply index.html]: - change div tag to h1 tag

Comment (Regiss05) [reply index.html]: I've done it

**Pull request merged**